utils/listable/lib/listable-format.js

```diff
@@ -37,12 +37,13 @@ function parseViewOptions(options) {
     showNDJSON: options.ndjson,
     showParseable: options.parseable,
     isTopological: options.toposort,
+    forceLocal: options.forceLocal,
```

Comment:
I'm not a fan of `--force-local` in the first place, and this isn't appropriate, as none of the listable commands accepts `--force-local` (nor should they).